File: Ruby/ITP1_05_A.rb
while true
  a = gets.split(" ").map(&:to_i)
  if a[0] == 0 && a[1] == 0 then
    break
  end
  a[0].times{|i|
    a[1].times{|j|
      print "#"
    }
    puts ""
  }
  puts ""
end

File: Python3/ITP1_05_A.py
while True:
    n = [int(i) for i in input().split()]
    if n[0] == 0 and n[1] == 0:
        break
    for i in range(0, n[0]):
        for j in range(0, n[1]):
            print("#", end=""),
        print()
    print()
